README.md

```diff
@@ -219,6 +219,47 @@ let () = printGithubRepos ()
 ## Node-specific
 
 #### Read lines from a text file
+
+```ml
+let () =
+    Node_fs.readFileSync "README.md" `utf8
+    |> Js.String.split "\n"
+    |> Array.iter Js.log
+```
+
 #### Read and parse a JSON file
+Uses [bs-json](https://github.com/BuckleTypes/bs-json)
+```ml
+let decodeName text =
+  (* parse JSON *)
+  match Js.Json.parseExn text with
+    | obj -> 
+        (* decode the field [name] *)
+        Json.Decode.(field "name" string obj)
+    | exception _ -> failwith ("Error parsing: " ^ text)
```

Comment:
Do you mean getting rid of the `| exception _ -> failwith ("Error parsing: " ^ text)
` branch ?